twitchio/http.py

```diff
@@ -334,6 +334,12 @@ async def get_bits_board(
     async def get_cheermotes(self, broadcaster_id: str):
         return await self.request(Route("GET", "bits/cheermotes", "", query=[("broadcaster_id", broadcaster_id)]))
 
+    async def get_channel_emotes(self, broadcaster_id: str):
+        return await self.request(Route("GET", "chat/emotes", "", query=[("broadcaster_id", broadcaster_id)]))
```

Comment:
as mentioned, broadcaster_id should be optional here, and the query parameter should be removed if it is `None`